### PR TITLE
Update issue and PR templates

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: feature/<issue-num>-<description>
      Example: feature/123-my-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from the **_master_** branch.

- [x] The **_base(target)_** branch for this pull request is the **_master_**.
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

- [x] Pull request title matches the title of the linked issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_master_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-todo-issue-template.yml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    render: markdown
+
+
+- type: checkboxes
+  attributes:
+    label: Perform Release ToDo List
+    description: Complete the following items to perform a release.
+    options:
+    - label: "Title updated by replacing the '**_\<TODO: release type here\>_**' section with the type of release."
+    - label: "Title updated by replacing the '**_\<TODO: release version number here\>_**' section with the version number."
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/release-todo-issue-template.yml:25]: Please remove the backslash symbols `\` from these 2 lines.

```suggestion
    - label: "Title updated by replacing the '**_<TODO: release type here>_**' section with the type of release."
    - label: "Title updated by replacing the '**_<TODO: release version number here>_**' section with the version number."
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
